packages/jaeger-ui/src/utils/interval-calculator.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const time = new (function() {
+  this.second = 1000;
+  this.minute = 60 * this.second;
+  this.hour = 60 * this.minute;
+  this.day = 24 * this.hour;
+})();
+
+const intervalMin = [
```

Comment:
ranges are normally expressed as `[min, max]`, why is this flipped?

Reply:
where is it expressed like this?